tests/test_ppdbBigQuery.py

```diff
@@ -0,0 +1,114 @@
+# This file is part of dax_ppdb.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (http://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import gc
+import os
+import shutil
+import tempfile
+import unittest
+from typing import Any
+
+from lsst.dax.apdb import ApdbConfig
+from lsst.dax.apdb.sql import ApdbSql
+from lsst.dax.ppdb import PpdbConfig
+from lsst.dax.ppdb.bigquery import PpdbBigQuery
+from lsst.dax.ppdb.tests import PpdbTest
+
+try:
+    import testing.postgresql
+except ImportError:
+    testing = None
+
+TEST_SCHEMA = os.path.join(os.path.abspath(os.path.dirname(__file__)), "config/schema.yaml")
+
+
+class SqliteTestCase(PpdbTest, unittest.TestCase):
+    """A test case for the PpdbBigQuery class using a SQLite backend."""
+
+    def setUp(self) -> None:
+        self.tempdir = tempfile.mkdtemp()
+        self.apdb_url = f"sqlite:///{self.tempdir}/apdb.sqlite3"
+        self.ppdb_url = f"sqlite:///{self.tempdir}/ppdb.sqlite3"
+
+    def tearDown(self) -> None:
+        shutil.rmtree(self.tempdir, ignore_errors=True)
+
+    def make_instance(self, **kwargs: Any) -> PpdbConfig:
+        """Make config class instance used in all tests."""
+        bq_config = PpdbBigQuery.init_database(db_url=self.ppdb_url, schema_file=TEST_SCHEMA, **kwargs)
+        bq_config.directory = self.tempdir
+        bq_config.delete_existing = True
```

Comment:
These should be arguments to `PpdbBigQuery.init_database`

Reply:
> init_database is not a polymorphic method, it is supposed to be used on a specific class.

I agree and was going to clean this up on the other ticket related to BQ configuration (DM-52460), which I am planning to work on next. `PpdbBigQuery` needs its own, separate initialization method. Do you want me to do some of this work on this branch instead?

Reply:
> Up to you.

Okay, I will clean that up on the other ticket - this branch already has enough changes on it.